docs/deb-package-install.adoc

```diff
@@ -10,10 +10,10 @@ sudo apt install ./aktualizr.deb
 
 The debian package will install, enable, and start an `aktualizr` systemd service immediately after it's installed. However, there are some configuration steps that should be taken before the service starts. To use aktualizr with a server (i.e. https://github.com/advancedtelematic/ota-community-edition/[OTA Community Edition] or https://docs.atsgarage.com[HERE OTA Connect]), you will need to download the provisioning credentials file provided by the server and place it at `/var/sota/sota_provisioning_credentials.zip`.
 
-Additionally, if you wish to pass extra command line arguments to aktualizr, you can do so by creating `/usr/lib/sota/sota.env` to set the `AKTUALIZR_SECONDARY_CONFIG` environment variable. For example, to enable a secondary ECU with a config file at `/var/sota/sec_ecu.json`, the contents of the file would be:
+Additionally, if you wish to pass extra command line arguments to aktualizr, you can do so by creating `/usr/lib/sota/sota.env` to set the `AKTUALIZR_SECONDARY_CONFIG` environment variable. For example, to set up the log level to debug:
 
 ----
-AKTUALIZR_SECONDARY_CONFIG="--secondary-config /var/sota/sec_ecu.json"
+AKTUALIZR_SECONDARY_CONFIG="--loglevel 0"
```

Comment:
Rather a strange name for the variable that adds an arbitrary configuration.

Reply:
I assume that is a typo or leftover from debugging?

Reply:
Yes you're right, thanks! In fact in meta-updater, we use `AKTUALIZR_CMDLINE_PARAMETERS`: https://github.com/advancedtelematic/meta-updater/blob/c035200bda65f96880cb1663a5841faeb4470861/recipes-sota/aktualizr/files/aktualizr.service#L10